ui/app/AppLayouts/Chat/panels/communities/BackUpCommuntyBannerPanel.qml

```diff
@@ -20,7 +20,7 @@ Rectangle {
     anchors.rightMargin: Style.current.padding
     border.color: Style.current.border
     radius: 16
-    color: Style.current.transparent
+    color: "transparent"
```

Comment:
Just to know, should we use `"transparent"` instead `Style.current.transparent` always?

Reply:
I think it makes sense yea. Transparent is always the same no matter what theme + `Style.current` is legacy anyways and will be replaced by `Theme` longterm